user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+
+## First: Don't panic
+
+The first thing you should do is: keep calm. We're here to help and we're confident that we can bring it back online together with you!
+
+## Prerequisites
+
+Please make sure that you have root access on all machines Travis CI Enterprise is running on. This is crucial because we will ask you to perform operations where you either need to be root or you need to have the appropriate `sudo` privileges. If you don't have direct root privileges, please make sure that a person who has is available.
+
+## Coordinating with Support
+
+We are here to help troubleshoot and fix any problems that may arise with your Travis CI Enterprise installation. For efficiency, though, we would request you minimize making changes to your setup that are not directly recommended by support, or document these steps and notify support accordingly.
+
+## Terminology
+
+To make sure that there are no confusions, we'd like to define a couple of terms:
+
+- `Platform machine`: The machine that runs most of the Travis web components. This is the machine your domain is pointing to.
+- `Worker machine`: The worker machine(s) run your builds.
+
+## Backup your encryption key
+
+This key is a very important building block in Travis CI Enterprise. It is used to encrypt certain parts of your production database. _Without this key the information in the database is not recoverable._ To make sure that the database remains accessible we ask you to make a backup of this key by following the steps below:
+
+- open a ssh connection to the platform machine
+- run `travis bash`. This will open a bash session with `root` privileges into the Travis container.
+- Then run `cat /usr/local/travis/etc/travis/config/travis.yml | grep -A1 encryption:`. Create a backup of value getting printed on the screen by either  writing it down on a piece of paper or storing it on a different computer.
+
+## Create a backup of the data directories
+
+The data directories are located on the platform machine and get mounted into the Travis container. In there you'll find files from RabbitMQ, Postgres, Slanger and Postgresql.
+
+These files are located at `/var/travis`. Please run `sudo tar -czvf travis-enterprise-data-backup.tar.gz /var/travis` to create compressed archive from this folder. After this has finished, copy this file off the machine to a secure location.
+
+## Pull a Support bundle
+
+Please head over to https://<your domain>:8800/support and create a support bundle. If you're writing us, please include that file. It'll include all important log files which will help us to diagnose the problem.
+
+Also please include the worker log files as well. They're located on your worker machine(s) at `/var/log/upstart/travis-worker.log`. If you run multiple worker machines, please grab this file from all machines.
+
+## What exactly is not running?
```

Comment:
Call this something like

## What information should I include in my support request? 

or even better, if it is a numbere step:

## Write a support request

Reply:
And then every subsequent header should be `###`

Reply:
I like this idea! And, a final step can be "please document any changes you make while working with support...", etc